docs/source/coremltools.models.rst

```diff
@@ -0,0 +1,60 @@
+Models
+==========================
+
+MLModel
+-------------------------------
+
+.. automodule:: coremltools.models.model
+   :members:
+
+
+feature\_vectorizer
+---------------------------------------------
+
+.. automodule:: coremltools.models.feature_vectorizer
+   :members:
+
+
+nearest\_neighbors
+---------------------------
+
+.. automodule:: coremltools.models.nearest_neighbors.builder
+   :members:
+
+
+neural\_network
+-------------------------------------------------
+
+.. toctree::
+   :maxdepth: 1
+   
+   coremltools.models.neural_network
+
+
+array\_feature\_extractor 
```

Comment:
Probably want to move this section to the top. I believe it's alphabetized besides that.